File: code/e7234.cpp
#include <iostream>
using namespace std;
int main()
{
    int i,n, a[1000],dem;
    cin>>n;
    for (i=1; i<=n; i++) cin>>a[i];
    dem=0;
    for (i=1; i<=n; i++) 
        if (a[i]%2==0) {
            dem++;
        }
    cout<<dem;
    return 0;
}

File: code/e7234.pas
var i, n, dem: integer;
    a : array[1..1000] of integer;
BEGIN
    readln(n);
    for i:= 1 to n do read(a[i]);
    dem:=0;
    for i:= 1 to n do
        if a[i] mod 2 = 0 then inc(dem);
    write(dem)
END.